uml4net.Extensions.Tests/PropertyExtensionsTestFixture.cs

```diff
@@ -151,6 +151,61 @@ public void Verify_that_QueryCSharpTypeName_returns_expected_Result()
             Assert.That(name.QueryCSharpTypeName(), Is.EqualTo("string"));
         }
 
+        [Test]
+        public void Verify_that_QueryCSharpTypeName_returns_expected_Result_when_Custom_mappings_are_added()
```

Comment:
please add an example in the code-gen project how this is supposed to be used. Have a look at the SysML2 model that contains datatypes that are not yet supported.

Reply:
done